content/python/concepts/built-in-functions/terms/breakpoint/breakpoint.md

```diff
@@ -34,3 +63,9 @@ breakpoint()
 
 print("Welcome back to Codecademy!!")
 ```
+
+Let's take a look at what happens when you run this code in your terminal:
+
+![Screenshot of terminal showing the breakpoint() formula](Screenshot-Terminal-Breakpoint.png)
```

Comment:
Images should be in the Docs\media folder.

Reply:
@SSwiniarski I did so in my last commit. One thing I'm not sure about is the image link. Using ../../media/python-built-in-functions-breakpoint-screenshot.png didn't work, so I looked at other .md files that contain an image in the media folder and used the link they're using (but adapted it to my filename). Could you check if that works? In the Codecademy Docs style guide I couldn't find anything on images.

Reply:
That should work.